articles/openshift/oshift-how-v4-update.md

```diff
@@ -62,11 +62,11 @@ We require customers to remain no further than one (1) minor version behind the
 
 For some additional information regarding upgrading a cluster, see the following OpenShift documentation:
 
-- <https://docs.openshift.com/container-platform/4.7/updating/updating-cluster-between-minor.html>
+- <https://docs.openshift.com/container-platform/4.10/updating/understanding-openshift-updates.html>
 
-- <https://docs.openshift.com/container-platform/4.7/updating/updating-cluster.html>
+- <https://docs.openshift.com/container-platform/4.10/updating/updating-cluster.html>
```

Comment:
Link leads to 404 page.
Would https://docs.openshift.com/container-platform/4.10/updating/updating-cluster-within-minor.html be suitable?